main.go

```diff
@@ -164,10 +203,117 @@ func main() {
 		setupLog.Info("starting http server", "address", address)
 		log.Fatal(http.ListenAndServe(address, h))
 	}()
+
+	var mutex *sync.Mutex
+
+	k8sClient, err := getK8sClientSet()
+	if err != nil {
+		setupLog.Error(err, "Cannot get the k8s client set")
+		os.Exit(1)
+	}
+
+	stop := make(chan struct{})
+	defer close(stop)
+
+	operatorNamespace, err := getOperatorNamespace(Config.LocalRun)
+	if err != nil {
+		setupLog.Error(err, "Cannot get the operator namespace")
+		os.Exit(1)
+	}
+	setupLog.V(1).Info("operator namespace found", "operatorNamespace", operatorNamespace)
+
+	configMapName, err := getConfigMapName(k8sClient, operatorNamespace, Config.LocalRun)
+	if err != nil {
+		setupLog.Error(err, "operator namespace", operatorNamespace)
+		os.Exit(1)
+	}
+
+	currentConfigMap, err := k8sClient.CoreV1().ConfigMaps(operatorNamespace).Get(context.TODO(), configMapName, metav1.GetOptions{})
+	if err != nil {
+		setupLog.Error(err, "Unable to get ConfigMap", "namespace", operatorNamespace)
+		os.Exit(1)
+	}
+	setupLog.V(1).Info("operator configmap found", "operatorNamespace", operatorNamespace, "configmap name", configMapName)
+
+	mutex = &sync.Mutex{}
+	go watchers.WatchForChanges(k8sClient, Config.LocalRun, operatorNamespace, configMapName, "VERBOSITY", currentConfigMap.Data["VERBOSITY"], mutex)
+
 	setupLog.Info("starting manager")
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
 		setupLog.Error(err, "problem running manager")
 		os.Exit(1)
 	}
+}
+
+func getK8sClientSet() (*kubernetes.Clientset, error) {
+	if Config.LocalRun {
```

Comment:
The manual stop&restart is not related to the LocalRun option. 
The LocalRun option discriminates between in cluster and out of the cluster run because the kubernetes ClientSet must be retrieved differently int the two cases (otherwise all the ClientSet will be `nil`). Is there any way to implicitly get the in/out cluster run? Or am I misunderstanding something?

Reply:
How about do nothing if `Config.LocalRun` is true ?
In dev-mode the value of the verbosity can be provided by the CLI, unlike in a production env.
We can agree that for log-level change, the developer will have to stop & start after setting new value.
Creating the watcher will be done only for production (non local run).

Then we won't have to support dev-mode code in production env.